lib/providers/content_provider.dart

```diff
@@ -0,0 +1,352 @@
+      response: queryResult.data != null
+          ? Query$FindCompanyTypes.fromJson(
+              queryResult.data!,
+            ).findCompanyTypes
+          : null,
+    );
+    if (result.response != null) _setCompanyTypeOptions(result.response!);
+    return result;
+  }
+
+  Future<OperationResult<List<CompanyStage>>> findCompanyStages() async {
+    final QueryResult queryResult = await asyncQuery(
+      queryOptions: QueryOptions(
+        document: documentNodeQueryFindCompanyStages,
+        fetchPolicy: FetchPolicy.cacheFirst,
+      ),
+    );
+    final result = OperationResult(
+      gqlQueryResult: queryResult,
+      response: queryResult.data != null
+          ? Query$FindCompanyStages.fromJson(
+              queryResult.data!,
+            ).findCompanyStages
+          : null,
+    );
+    if (result.response != null) _setCompanyStageOptions(result.response!);
+    return result;
+  }
+
+  Future<OperationResult<List<EducationLevel>>> findEducationLevels() async {
+    final QueryResult queryResult = await asyncQuery(
+      queryOptions: QueryOptions(
+        document: documentNodeQueryFindEducationLevels,
+        fetchPolicy: FetchPolicy.cacheFirst,
+      ),
+    );
+    final result = OperationResult(
+      gqlQueryResult: queryResult,
+      response: queryResult.data != null
+          ? Query$FindEducationLevels.fromJson(
+              queryResult.data!,
+            ).findEducationLevels
+          : null,
+    );
+    if (result.response != null) _setEducationLevelOptions(result.response!);
+    return result;
+  }
+
+  Future<OperationResult<List<PresetGender>>> findPresetGenders() async {
+    final QueryResult queryResult = await asyncQuery(
+      queryOptions: QueryOptions(
+        document: documentNodeQueryFindGenders,
+        fetchPolicy: FetchPolicy.cacheFirst,
+      ),
+    );
+    final result = OperationResult(
+      gqlQueryResult: queryResult,
+      response: queryResult.data != null
+          ? Query$FindGenders.fromJson(
+              queryResult.data!,
+            ).findGenders
+          : null,
+    );
+    if (result.response != null) _setPresetGenderOptions(result.response!);
+    return result;
+  }
+
+  Future<OperationResult<Query$FindAllOptionsByType>>
+      findAllOptionsByType() async {
+    final QueryResult queryResult = await asyncQuery(
+      queryOptions: QueryOptions(
+        document: documentNodeQueryFindAllOptionsByType,
+        fetchPolicy: FetchPolicy.cacheFirst,
+      ),
+    );
+    final result = OperationResult(
+      gqlQueryResult: queryResult,
+      response: queryResult.data != null
+          ? Query$FindAllOptionsByType.fromJson(
+              queryResult.data!,
+            )
+          : null,
+    );
+    if (result.response != null) {
+      _setAllContentOptions(
+        companyStages: result.response?.findCompanyStages
+            .map((e) =>
+                Query$FindCompanyStages$findCompanyStages.fromJson(e.toJson()))
```

Comment:
This `fromJson` `toJson` feels wrong but I didn't know how else to assign a variable of type  `Query$FindAllOptionsByType$findIndustries` to `Query$FindIndustries$findIndustries`.

Maybe a bit of premature optimization here trying to do this all in one request rather than the 10+ requests to do everything separately, as we will probably only do this once on the app.